src/app/portlets/content-types/content-types.module.ts

```diff
@@ -103,6 +105,8 @@ import { DotBaseTypeSelectorModule } from '../../view/components/dot-base-type-s
         ConfirmDialogModule,
         ContentTypesRoutingModule,
         DialogModule,
+        DotIconModule,
```

Comment:
Do we need it here if we already have it in the `app.module`?

Reply:
yes it is needed